### DISCUSSION TITLE
Regular package updates

### DISCUSSION THREAD

**Discussion by LordHelmchen666** [JoomCoder-com/JoomCCK#182] [General]
Could you  please provide regular package updates when you fixed things. So we could continue testing and not report bugs again that already fixed. Would be great!

**Comment by Giorgi625** [comment]
Agree, we still have not got updates for january fixes.

**Comment by andiamine** [comment]
Accepted we are going to release a new update once we fix some issues.

**Reply by LordHelmchen666** [reply]
@andiamine Thank you so much for your work. Could you please fix this two issues https://github.com/JoomCoder-com/JoomCCK/issues/186 and https://github.com/JoomCoder-com/JoomCCK/issues/171 before you release new package?
These are just typos, but they breaks my site. So i don't have to fix it with every update manually.
